imports/plugins/core/taxes/server/methods/methods.js

```diff
@@ -108,7 +108,7 @@ export const methods = {
     check(cartId, String);
     check(options, {
       taxRatesByShop: Match.OneOf(undefined, null, Object),
-      itemsWithTax: [Object],
+      itemsWithTax: [Match.OneOf(Object, undefined)],
```

Comment:
fyi you can do
`Match.Optional(Pattern)` for `Match.OneOf(Pattern, undefined)`
and
`Match.Maybe(Pattern)` for `Match.OneOf(Pattern, undefined, null)`